magni_gazebo/scripts/motion_vectors.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import rospy
+from raspicam_node.msg import MotionVectors
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+from skvideo.motion import blockMotion
```

Comment:
im trying to run the code, but the dependency skvideo.motion is missing. Can you provide instructions how to install this?

Reply:
`pip install scikit-video`